kernel/aster-nix/src/process/signal/sig_queues.rs

```diff
@@ -131,6 +131,15 @@ impl SigQueues {
         None
     }
 
+    /// Returns whether the sigqueue has some pending signal which is not blocked
+    pub fn has_pending_signal(&self, blocked: SigMask) -> bool {
```

Comment:
sigqueue -> `SigQueues`, pending signal which is -> pending signals which are. I think `signal` in the function signature might also need to be plural to be consistent.